pkg/kubelet/apis/config/types.go

```diff
@@ -62,6 +62,8 @@ const (
 type KubeletConfiguration struct {
 	metav1.TypeMeta
 
+	// rootDir is the directory path to place kubelet files (volume mounts, etc).
+	RootDir string
```

Comment:
are all of these filepaths allowed to be relative or must they be absolute?

Reply:
I think it's fine to allow them to be relative, if only because it keeps the file path treatment consistent across config (either make it absolute or it's relative to the file, or specify via flag for legacy relative to pwd behavior).

Reply:
do they get made absolute before they get passed down into the kubelet? It wouldn't surprise me if some/most of the volume operations required absolute base paths

Reply:
The values from config are always made absolute based on the location of the config file.
Then flags override verbatim, which for relative paths relies on the default behavior of being relative to the working dir. 

I wouldn't want to change the flag behavior for fear of breaking something, and since the paths from the config file are made absolute before being used, relative paths shouldn't be an issue there.